paper-server/patches/sources/net/minecraft/world/level/GameRules.java.patch

```diff
@@ -14,7 +14,7 @@
 +
      public static final int DEFAULT_RANDOM_TICK_SPEED = 3;
      static final Logger LOGGER = LogUtils.getLogger();
-     private static final Map<GameRules.Key<?>, GameRules.Type<?>> GAME_RULE_TYPES = Maps.newTreeMap(Comparator.comparing(entry -> entry.id));
+     public static final Map<GameRules.Key<?>, GameRules.Type<?>> GAME_RULE_TYPES = Maps.newTreeMap(Comparator.comparing(entry -> entry.id));
```

Comment:
Maybe https://github.com/PaperMC/Paper/blob/main/CONTRIBUTING.md#access-transformers would be better here?

Reply:
That is an AT.